src/lib/utils/sanitizeTiptapContent.ts

```diff
@@ -0,0 +1,24 @@
+import sanitizeHtml from 'sanitize-html';
+
+export const sanitizeTiptapContent = (content: string) => {
+	return sanitizeHtml(content, {
+		allowedClasses: {
+			'*': ['*']
```

Comment:
this seems like [clickjacking](https://owasp.org/www-community/attacks/Clickjacking) just waiting to happen - no PoC I have in mind, though.

Reply:
add a boolean or so, `trusted`, that allows custom classes and img for articles.

Reply:
resolved with 930cd3278af039c66085b0428697c497d7573b76